packages/gatsby-plugin-sharp/src/gatsby-node.js

```diff
@@ -131,16 +126,9 @@ exports.onPostBootstrap = async ({ reporter, cache, store }) => {
 
 if (coreSupportsOnPluginInit) {
   // to properly initialize plugin in worker (`onPreBootstrap` won't run in workers)
-  if (_CFLAGS_.GATSBY_MAJOR === `4`) {
-    exports.onPluginInit = async ({ actions }, pluginOptions) => {
-      setActions(actions)
-      setPluginOptions(pluginOptions)
-    }
-  } else {
-    exports.unstable_onPluginInit = async ({ actions }, pluginOptions) => {
-      setActions(actions)
-      setPluginOptions(pluginOptions)
-    }
+  exports.onPluginInit = async ({ actions }, pluginOptions) => {
+    setActions(actions)
+    setPluginOptions(pluginOptions)
   }
 }
```

Comment:
This file can only contain
```
exports.onPluginInit = async ({ actions }, pluginOptions) => {
    setActions(actions)
    setPluginOptions(pluginOptions)
  }
```

Reply:
Do you removing `if (coreSupportsOnPluginInit) {` check ?

Reply:
Yes

Reply:
👍